README.md

```diff
@@ -115,14 +115,29 @@ The generator adds ShopifyApp and the required initializers to the host Rails ap
 After running the `install` generator, you can start your app with `bundle exec rails server` and install your app by visiting localhost.
 
 
-### Home Controller Generator
+### Root Controller Generator
 
 ```sh
-$ rails generate shopify_app:home_controller
+$ rails generate shopify_app:root_controller
 ```
 
-This generator creates an example home controller and view which fetches and displays products using the Shopify API.
+This generator creates an unauthenticated root controller your app can default to.
 
+### Authenticated Controller Generator
+
+```sh
+$ rails generate shopify_app:authenticated_controller
+```
+
+This generator creates an example authenticated controller to protect resources with.
```

Comment:
Is this controller an example, or is it something they'll likely want to keep as-is?

Reply:
Hmm good point. I think the word "example" can be omitted here.